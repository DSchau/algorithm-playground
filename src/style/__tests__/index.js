import * as STYLE from '..';

test('it has expected exports', () => {
  expect(Object.keys(STYLE)).toMatchSnapshot();
});
