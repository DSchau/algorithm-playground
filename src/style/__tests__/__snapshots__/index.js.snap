// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`it has expected exports 1`] = `
Array [
  "FADE_IN",
  "SCALE_IN",
  "SCALE_OUT",
  "SLIDE_UP",
  "SMALL",
  "MEDIUM",
  "LARGE",
  "SANS_SERIF",
  "SERIF",
  "THEME",
  "Z_INDEX_BASE",
  "Z_INDEX_PREVIEW_CONTENT",
  "Z_INDEX_ABOVE_CONTENT",
  "Z_INDEX_SUPER",
]
`;
